bokeh/models/map_plots.py

```diff
@@ -178,10 +178,10 @@ def _check_missing_google_api_key(self):
 
     background_fill_alpha = Override(default=0.0)
 
-    api_key = NonNullable(Base64String, help="""
+    api_key = NonNullable(Bytes, help="""
     Google Maps API requires an API key. See https://developers.google.com/maps/documentation/javascript/get-api-key
     for more information on how to obtain your own.
-    """)
+    """).accepts(String, lambda val: val.encode("utf-8"))
```

Comment:
It will be base64 encoded as before, and it will be binary when available. However, I intend to add a threshold for small binary buffers, so that we don't send a few bytes a separate messages. This way keys will end up always base64 encoded.

Reply:
Ah right because all bytes are when not buffered, perfect 👍